cmd/istio-cni/redirect.go

```diff
@@ -225,6 +225,12 @@ func NewRedirect(annotations map[string]string) (*Redirect, error) {
 			"excludeOutboundPorts", isFound, valErr)
 		return nil, valErr
 	}
+	// Add 15090 to sync with non-cni injection template
+	redir.excludeInboundPorts = strings.TrimSpace(redir.excludeInboundPorts)
+	if len(redir.excludeInboundPorts) > 0 && redir.excludeInboundPorts[len(redir.excludeInboundPorts)-1] != ',' {
+		redir.excludeInboundPorts = redir.excludeInboundPorts + ","
+	}
+	redir.excludeInboundPorts = redir.excludeInboundPorts + "15090"
```

Comment:
Why add it here instead of https://github.com/istio/istio/blob/a671e62bd1ca3e6fd09bd0fbe03545c136b0f3f5/manifests/charts/istio-control/istio-discovery/files/injection-template.yaml#L406? This is where the other port, 15020, is defined?

Reply:
That may be my fault, I only saw the other annotation (which makes changes directly to the proxy sidecar) and assumed there was no similar way to pass that to the CNI pod. 

I can create a PR for the other, that would be a lower risk fix

Reply:
Oh, more ports?

Reply:
so we need another statusPort here in cni
and btw: where is `podRedirectAnnot` used?